mautic/mautic
# Fix 10455 edit code is broken - rebased on 4 1

closes #10455

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | yes
| Deprecations?                          |  no
| BC breaks? (use the c.x branch)        | no
| Automated tests included?              | N/A
| Issue(s) addressed                     | Fixes #10455

#### Description:

This fixes an issue where the code editor modal did not open **again** when the grapesjs builder was closed.

#### Steps to test this PR:

1. Load up [this PR](https://mautibox.com)
2. See steps in the [issue](https://github.com/mautic/mautic/issues/10455)


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10564"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>


This PR is the same as https://github.com/mautic/mautic/pull/10564 but based on 4.1 branch

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10689"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
